src/pybind/mgr/dashboard/frontend/src/app/shared/directives/tab-stateful.directive.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { TabStatefulDirective } from './tab-stateful.directive';
+
+describe('TabStatefulDirective', () => {
```

Comment:
Could you please add some tests to this shared direktive? As it is used in multiple places it would be nice to have it well tested.